cloud.foundry.cli/src/main/java/cloud/foundry/cli/services/CreateController.java

```diff
@@ -74,6 +78,7 @@ public Integer call() throws Exception {
 
         @Override
         public Integer call() throws Exception {
+            Log.info("Create service(s) running...");
```

Comment:
"Creating service(s)..." The three dots imply that there will be a "reply" soon, e.g., "Created services". This is the case here.

Reply:
Done